File: 백준/Bronze/10162. 전자레인지/전자레인지.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./example.txt";
let input = fs.readFileSync(filePath).toString().split("\n");
input = parseInt(input);

let times = [300, 60, 10];
let count = [];
for (let time of times) {
  quotient = parseInt(input / time);
  input = input - time * quotient;
  count.push(quotient);
}
console.log(input ? -1 : count.join(" "));

File: 백준/Bronze/10162. 전자레인지/전자레인지.py
n = int(input())

times = [300, 60 ,10]

if n%times[2] !=0: 
    print(-1)
    exit()

a = n // times[0]
n %= times[0] 

b = n // times[1]
n %= times[1] 

c = n // times[2]
n %= times[2] 

print(a, b, c)
